packages/catalog-process/src/services/catalogService.ts

```diff
@@ -3123,7 +3144,7 @@ export async function createOpenApiInterfaceByTemplate(
           contentType,
           checksum,
           serverUrls,
```

Comment:
Here don't you need to pass also the `serverUrls` that the user is passing in the interface data? 

```suggestion
          serverUrls: eserviceInstanceInterfaceData.serverUrls
```

Reply:
👍🏻 as discuss offline, this refactor was included in this commit [c079916](https://github.com/pagopa/interop-be-monorepo/pull/1643/commits/c079916caed33e0e59df7371ab64cddd812a2a59).